app/src/components/main/attributeList/editableAttribute/attributeEditor/attributeEditor.jsx

```diff
@@ -180,7 +190,11 @@ export class AttributeEditor extends Component {
               minLength={1}
               attributes={attributes}
               attributeComparator={this.byValueComparator}
-              getURI={valueURLCreator(projectId, this.state.key)}
+              getURI={
+                launchId
```

Comment:
Is the `launchId` needed for every time where this component used?
I think it will be better to use `launchId` on the top level component.